doc/en/user/source/extensions/csw-iso/queryables.rst

```diff
@@ -0,0 +1,14 @@
+CSW ISO Metadata Profile Queryables
+===================================
+
+Mapping
+~~~~~~~
+The ISO Metadata standard (see `OGC Implementation Specification 07-045 <http://www.opengeospatial.org/standards/specifications/catalog>`_) specifies a list of so called 'queryables'. These are simple property names such as 'Title' or 'Abstract' that can be used in filters and will automatically be translated to a corresponding XPath.
+
+In some case it might be required to have an alternative mapping of queryables (to other than the default XPaths). For instance when records represent services rather than data, the corresponding XPaths are different.
```

Comment:
```suggestion
In some case it might be required to have an alternative mapping of queryables (to a different, non-default XPath). For instance when records represent services rather than data, the corresponding XPaths are different.
```